src/members/MemberProfile.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+
+
+export default function MemberProfile() {
```

Comment:
Can rename this component to Member

Reply:
@meenakshi-dhanani . Later once the Organisation, Program and Mentorship Relation functionalities are completed, the User and Member area will have User Portfolio and Member Portfolio that show programs that the user and member have been involved (like the dashboard). So, I think we should keep the name MemberProfile since later we'll have MemberPortfolio. What do you think?